.changeset/sour-plums-pull.md

```diff
@@ -0,0 +1,15 @@
+---
+"wrangler": patch
+---
+
+feat: resolve imports in a more node-like fashion for packages that do not declare exports
```

Comment:
```suggestion
fix: resolve imports in a more node-like fashion for packages that do not declare exports
```

This is really a bug fix; and in any case if it was a feature we'd need a minor version bump here.